src/main/java/org/advent/daySeven/Type.java

```diff
@@ -0,0 +1,22 @@
+package org.advent.daySeven;
+
+public enum Type {
+
+    HIGH_CARD(1),
+    ONE_PAIR(2),
+    TWO_PAIR(3),
+    THREE_OF_A_KIND(4),
+    FULL_HOUSE(5),
+    FOUR_OF_A_KIND(6),
+    FIVE_OF_A_KIND(7);
```

Comment:
You are mimicing ordinals here, there is no need to have these values asigned at all. Google enum ordinals :)